static/app/views/monitors/components/timeline/gridLines.tsx

```diff
@@ -23,6 +24,10 @@ interface Props {
    * Enable the timeline cursor
    */
   showCursor?: boolean;
+  /**
+   * Render sentry service incidents as an overlay
+   */
+  showIncidents?: boolean;
```

Comment:
hmm are there instances where we actually don't want to show the incidents besides `isLoading`? I wonder if we should just have a `isLoading` here which disables/enables both `showCursor` and `showIncidents`

Reply:
yeah we don't show it on the visualization either

Reply:
ahh i see